src/contexts/LayoutContext.tsx

```diff
@@ -19,7 +21,9 @@ type Props = {
 };
 
 export const LayoutContextProvider = ({ children }: Props): JSX.Element => {
-  const [isPinnedMenuOpen, setIsPinnedMenuOpen] = useState<boolean>(true);
+  const { isMobile } = useMobileView();
+
+  const [isPinnedMenuOpen, setIsPinnedMenuOpen] = useState<boolean>(!isMobile);
```

Comment:
Not sure if it was the intent, but this value will only set the initial value of the state. If there is a change afterwards they will not be synched.

Reply:
@spaenleh 
Do u think normal user going to change screen size, It might be fixed using useeffect to synd the value 👍🏼

Reply:
Not sure it is needed, but I just wanted to make sure that it was omitted for a reason.